### ISSUE TITLE
Update user status to registered once email verification completes

### ISSUE THREAD

**Issue by gr4per** [OpusCapita/user#16]
Make sure user update event fires

**Comment by sureshvarman-cronj** [comment]
@Arne Once email verification is done, we are making the status to emailVerified, which is already place, but not as registered.

**Comment by gr4per** [comment]
@sureshvarman-cronj does this break the existing onboarding case and require changes in onboarding or are we still fine?

**Comment by gr4per** [comment]
@celber with this change, do we need updates in onboarding? pls check

**Comment by sureshvarman-cronj** [comment]
@Arne, https://github.com/OpusCapita/auth/issues/30, seems to be duplicate for me

**Comment by sureshvarman-cronj** [comment]
duplicate of https://github.com/OpusCapita/auth/issues/30
